knowledge-base/styling-weekend-days-radscheduler-monthview-net-maui.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Styling Weekend Days in RadScheduler Month View for .NET MAUI
+description: Learn how to style weekend days differently from weekdays in the MonthView of RadScheduler for .NET MAUI.
+type: how-to
+page_title: How to Style Weekend Days in RadScheduler Month View for .NET MAUI
+slug: styling-weekend-days-radscheduler-monthview-net-maui
+tags: calendar, .net maui, radscheduler, monthview, daystyleselector, weekend styling
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 7.0.0 | Telerik UI for .NET MAUI Scheduler | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova)| 
+
+## Description
+
+When using the `RadScheduler` in a .NET MAUI application, I wanted to style the weekend days differently from the weekdays in the `MonthView`. Initially, I thought to use the `SpecialSlotStyleSelector` for the `MonthViewDefinition`. I have used this property in Telerik Xamarin Scheduler, but it turns out this property is not available in Telerik MAUI Scheduler. Is there an alternative way to accomplish this?
+
+This KB article also answers the following questions:
+- How can I apply custom styling to weekend days in the Scheduler `MonthView`?
+- Is there a way to differentiate weekend and weekday styles in .NET MAUI Scheduler?
+- What property should be used for day styling in `MonthViewDefinition` of `RadScheduler`?
+
+## Solution
+
+To style days differently in the `MonthView` of `RadScheduler`, instead of using `SpecialSlotStyleSelector`, you should utilize the `DayStyleSelector`. This allows for custom styling of days, including differentiating weekends from weekdays.
```

Comment:
```suggestion
To style days differently in the `MonthView` of `RadScheduler` for .NET MAUI, instead of using `SpecialSlotStyleSelector`, you should utilize the `DayStyleSelector`. This allows for custom styling of days, including differentiating weekends from weekdays.
```